third_party/terraform/tests/resource_compute_disk_test.go.erb

```diff
@@ -828,4 +886,32 @@ resource "google_compute_disk" "foobar" {
   resource_policies = [google_compute_resource_policy.foo.self_link]
 }`, policyName, diskName)
 }
+
+func testAccComputeDisk_resourcePolicies_updated(diskName, policyName string) string {
+	return fmt.Sprintf(`
+data "google_compute_image" "my_image" {
+  family  = "debian-9"
+  project = "debian-cloud"
+}
+resource "google_compute_resource_policy" "bar" {
+  name = "%s"
+  region = "us-central1"
+  snapshot_schedule_policy {
+    schedule {
+      daily_schedule {
+        days_in_cycle = 3
```

Comment:
Since these values are all that changes, you can templatise them in the original config and supply different values in each stage of the test.